Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -67,6 +67,7 @@ def bump_cask_pr_args
   def bump_cask_pr
     args = bump_cask_pr_args.parse
 
+    odeprecated "brew bump-cask-pr --online" if args.online?
```

Comment:
It'd be nice if we could customize the deprecation message some more. Something like `--online is now on by default; use --no-audit to disable it` would be helpful here. Not blocking though. It looks like `deprecated` would need to be modified in that case though. Or maybe something like:

```rb
if args.online?
  deprecated "brew bump-cask-pr --online"
  puts "`--online` is now the default. Use `--no-audit` to avoid online checks."
end
```

Reply:
Comment this out for now and we'll add it on the next release.

Reply:
It's a bit tough though since there are other changes in the same file that need to be made in the same file. Probably better to just wait and do it all at once, right?

Reply:
@apainintheneck I don't think these changes to a developer command need to wait until the next major/minor release. The `odeprecated` is more debatable and I'm fine either way. If we do want to add it: it'd be nicer to comment it out now otherwise it'll likely be bumped to `odisabled` on the next major/minor release which isn't what we want.

Reply:
@razvanazamfirei Sorry for confusion: only the `odeprecated` line needs commented out.